contracts/FlowIDTableStaking.cdc

```diff
@@ -1405,8 +1424,10 @@ pub contract FlowIDTableStaking {
                 ?? panic("No moves pending list in account storage")
 
             // Reset the movesPendingList
-            let movesPendingList: {String: {UInt32: Bool}} = {}
-            FlowIDTableStaking.account.save<{String: {UInt32: Bool}}>(movesPendingList, to: /storage/idTableMovesPendingList)
+            var emptyMovesPendingList: {String: {UInt32: Bool}} = {}
+            FlowIDTableStaking.account.save<{String: {UInt32: Bool}}>(emptyMovesPendingList, to: /storage/idTableMovesPendingList)
```

Comment:
No need for explicit type argument, can use type inference. Maybe also remove in the rest of the contract while at it.

```suggestion
            FlowIDTableStaking.account.save(emptyMovesPendingList, to: /storage/idTableMovesPendingList)
```